frontend/glideinFrontendInterface.py

```diff
@@ -101,9 +131,34 @@ def __init__(self):
 
 
 class MultiExeError(condorExe.ExeError):
+    """Exception raised when an error occurs when executing a command and used to store multiple failures.
+
+     It helps to handle partial failures
+
+    Example:
+         >>> err1 = ExeError("Error 1")
+         >>> err2 = ExeError("Error 2")
+         >>> multi_err = MultiExeError([err1, err2])
+         >>> print(multi_err)
+         "Error 1\nError 2"
+    """
+
     def __init__(self, arr):
-        """
-        arr is a list of ExeError exceptions
+        """Exception raised for multiple ExeError exceptions.
```

Comment:
The docstring here still reads as being class-related rather than method-level. Should this section be removed since the class-level docstring was updated?

If removing, can we reuse some of the content here by moving it to the class-level docstring?